app/views/admin/organizations/show.html.slim

```diff
@@ -33,21 +33,13 @@ section.coop.details
     h1 = t('organization.media')
     .row = render "admin/media/index", owner: @org
 
-  section.block.notes.post-list
-    header.header-with-links
-      h1 = t('common.notes')
-      button.btn.btn-default.new-button = t("notes.new")
-    .notes-inner
-      = render @notes
-    
-    .hidden
-      .new-note
-        = render partial: 'admin/notes/note', locals: { note: @new_note }
+  section.block
+    = render "admin/notes/notes_section"
 
 = render "admin/media/modal"
 
 javascript:
   $(function() { 
-    new MS.Views.OrganizationView();
+    new MS.Views.NotableView({ el: 'coop' });
```

Comment:
Should there be a `.` before `coop`? I'm curious how this is even working currently... Please be sure to test all concrete instantiations when you abstract something out like this...